static/app/views/insights/agents/views/overview.tsx

```diff
@@ -75,6 +76,7 @@ function AgentsOverviewPage() {
   const showOnboarding = useShowOnboarding();
   const datePageFilterProps = limitMaxPickableDays(organization);
   const [searchQuery, setSearchQuery] = useLocationSyncedState('query', decodeScalar);
+  useDefaultToAllProjects();
```

Comment:
I was looking to see if there was a top-level component that we could shim this into and there's `ModulePageProviders`. It's where we set up the page providers, and the page filters provider is there, so I was wondering if you think it'd be cleaner to add some kind of inbetween layer to handle this effect in one place rather than having to manually call it in all of the views

Reply:
discussed in slack:
some of overview pages don't use module page providers, so I figured I'd just purposely put the hook on each overview page instead of putting it in the provider components and then plugging holes. also separates concerns a bit